src/parser/XMLParser.java

```diff
@@ -188,8 +188,12 @@ private static List<Location> setLocations(Element el, List<Clock> clocks, List<
             }
 
             Location  newLoc;
-            if (xyDefined) newLoc = Location.create(locName, invariants, isInitial, false, false, false, x, y);
-            else newLoc= Location.create(locName, invariants, isInitial, false, false, false, 0, 0);
+            if (xyDefined) {
+                newLoc = Location.create(locName, invariants, isInitial, false, false, false, x, y);
+            }
+            else {
+                newLoc = Location.create(locName, invariants, isInitial, false, false, false, 0, 0);
```

Comment:
You can now also just call `Location.create(locName, invariants, isInitial, false, false, false)` with your new `create` method.